testinfra/functional/test_tor_interfaces.py

```diff
@@ -5,6 +5,7 @@
 sdvars = pytest.securedrop_test_vars
 
 
+@pytest.mark.xfail
```

Comment:
The GET == 200 on the Source Interface over Tor has been an incredibly useful test. How about adding retries rather than expecting failure altogether?

Reply:
Do you have a suggestion for adding retries here? I figured pytest would have something for this but i'm having trouble finding anything native.

Reply:
I'll try a down-and-dirty for-loop that only fails after _n_ number of tries have been exhausted.

Reply:
I see what you mean about the flaky behavior—I've opened #2361 to drill down on the test, and I'm leaving it marked as XFAIL in this PR.